runtime/taskcontext.go

```diff
@@ -174,6 +176,10 @@ func (c *TaskContext) log(prefix string, a ...interface{}) {
 	if err != nil {
 		//TODO: Forward this to the system log, it's not a critical error
 	}
+
+	if c.LogEntry != nil {
```

Comment:
What are the use-cases for this?

If running a process you normally won't write logs using this method, but using the `LogDrain()`...
Ie if you have a subprocess running you set stdout and stderr to `TaskContext.LogDrain()`